fixtures/parser_tests.ts

```diff
@@ -184,24 +186,24 @@ function parserTests(parse: (file: string) => ParserResult) {
     it('finds test blocks within describe blocks', () => {
       const data = parse(`${fixtures}/dangerjs/travis-ci.example`);
       const descBlock = data.describeBlocks[1];
-      expect(descBlock.children.length).toBe(4);
+      expect(descBlock.children?.length).toBe(4);
 
       // check test blocks, including the template literal
-      const found = descBlock.children.filter(
+      const found = descBlock?.children?.map(b => b as ItBlock)?.filter(
```

Comment:
In my opinion we don't need a `?.` after `map`.

Reply:
👍 

How did you add these @rossknudsen?

Reply:
@seanpoulter if you mean this specific piece of code, it now looks like this:

```ts
// check test blocks, including the template literal
const found = descBlock.children
.map(b => b as NamedBlock)
.filter(
    b =>
    b.name === 'needs to have a PR number' ||
    b.name === 'does not validate without josh' ||
    b.name === 'does not validate when ${key} is missing'
);
expect(found.length).toBe(3);
```

I was just trying to assert that the type of the array elements were `NamedBlock`s (originally `ItBlock`s).  There is potentially another way to write this which is arguably more elegant.  Now that the `children` property is not potentially null, it is a lot clearer.

If you are talking more generally, you might need to give me some more context.